README.md

```diff
@@ -1,25 +1,330 @@
-# Cortex SDK
+# Cortex Click Node.js SDK
 
-An AI accelerated go-to-market (GTM) platform. Easily automate sales and marketing workflows from top of funnel content, to point of sale, support, and expansion.
+```
+npm install @cortexclick/cortex
```

Comment:
```
npm install --save @cortexclick/cortex
```

Reply:
I googled this because I don't know the answer, but it sounds like --save is the default behavior of npm install since v5: https://stackoverflow.com/questions/19578796/what-is-the-save-option-for-npm-install

Is there another reason to be explicit about this?

Reply:
nope, that is news to me!

![](https://media.tenor.com/3zOSoEyrSXwAAAAM/the-more-you-know.gif)